docs/source/start.rst

```diff
@@ -155,110 +165,64 @@ libffi. On a Debian-derivative system, this could be accomplished with
 
 To uninstall, remove the directory ``~/rigetti``.
 
+Getting Started
+~~~~~~~~~~~~~~~
+To get started using the SDK, you can either interact with the QVM and the compiler directly from the command line,
+or you can run them in server mode and use them with pyQuil.
 
-Using the SDK
```

Comment:
I'm confused why this is a diff, since I previously made this change and it's been merged.